daffodil-schematron/src/main/resources/iso-schematron-xslt2/readme.txt

```diff
@@ -0,0 +1,100 @@
+<h1>ISO SCHEMATRON 2010</h1>

```

Comment:
In the context of Daffodil, this file is a bit misleading. I understand including it purely for completeness (include all of it, to be clear nothing subset-wise is going on.) Is that the rationale for including this? 

If so then I would suggest the daffodil-schematron/README.md file contain explanation about including *everything* without exception, vs. picking and choosing just what is required.

Reply:
Yeah, this and the htm are in the official distribution of iso schematron, so I just left it in there without looking too hard.

Looking now I dont see any reason it has to remain.  IMO we can point to the online version of the htm and remove this txt.